### DISCUSSION TITLE
Feature request: Auto suggest language based on location or browser language

### DISCUSSION THREAD

**Discussion by ghost** [lloc/Multisite-Language-Switcher#206] [Ideas]
Hello,
I would like to request a feature for auto-suggesting visitors for their language based on their location or browser language.
Just like apple site, without auto-redirection.
or if there is a snippet for that please let me know :)

Thank you

**Comment by lloc** [comment]
There ware some activities ongoing ... like this here: https://github.com/oncleben31/Multisite-Language-Switcher-Theme/blob/master/index.php

I'm sure t could also be an add-on for MSLS.
